packages/storybook/src/react-components/button/button.stories.tsx

```diff
@@ -0,0 +1,270 @@
+import { LuxButton } from '@lux-design-system/components-react';
+import tokens from '@lux-design-system/design-tokens/dist/index.json';
+import type { Meta, StoryObj } from '@storybook/react';
+
+type Story = StoryObj<typeof meta>;
+
+const meta = {
+  title: 'React Components/Button',
+  id: 'react-components-button',
+  component: LuxButton,
+  subcomponents: {},
+  parameters: {
+    tokens,
+    tokensPrefix: 'utrecht-button',
+  },
+  argTypes: {
+    variant: {
+      description: 'Variant modifier',
+      control: 'select',
+      options: [undefined, 'primary', 'secondary', 'tertiary'],
+    },
+    size: {
+      description: 'Size modifier',
+      control: 'select',
+      options: [undefined, 'small'],
+    },
+    iconPosition: {
+      description: 'Icon Position modifier',
+      control: 'select',
+      options: [undefined, 'start', 'end'],
+    },
+    busy: {
+      description: 'Busy indicator',
+      control: 'boolean',
+    },
+    pressed: {
+      description: 'Pressed indicator',
+      control: 'boolean',
+    },
+    icon: {
+      description: 'Icon Node',
+      control: 'object',
+      table: {
+        type: {
+          summary: 'HTML Content',
+        },
+      },
+    },
+    label: {
+      description: 'Label Node',
+      control: 'object',
+      table: {
+        type: {
+          summary: 'HTML Content',
+        },
+      },
+    },
+    hint: {
```

Comment:
Hint kan weg, in de Utrecht CSS zet het een bepaalde class, maar ik zie er geen tokens voor.
Volgens mij is het om error/warning/etc-buttons te maken, maar deze tokens bestaan (nog) niet.